### PR TITLE
Add fail-safe for Fees to block the UI until loaded

### PR DESCRIPTION
**Pull request by gre**
In an edge-case scenario, it was possible to accidentally Send transactions without fees. This happen if you are fast enough to input everything and this before our API that fetches the fees respond and if you are not careful enough in what are in the fields.

This is now fixed by adding a fail safe that block the UI until fees load. Since we block the UI, we must provided a good feedback to the user on what's happening at any time. We use a spinner when fees are loaded (you likely won't see it, both because load is fast and also because we have a cache so fees get loaded only once). In case of a server failure, you would see an error as well but it's not blocking the Send feature to work. You can also interrupt the load at any time by clicking the input and typing yourself the fees.

capture d ecran 2018-09-25 a 17 13 45 capture d ecran 2018-09-25 a 17 10 27

^ once again, these 2 screen would be edge case and you likely don't have time to even see them.

(BTW, this Input loading will be useful if we need it somewhere else)

cc @dasilvarosa for a wording check (the error case)

### Type

edge case bug fix

### Context

was discussed internally a few weeks ago. thx @bscholving for brainstorming the UX.

### Parts of the app affected / Test plan

- because the work was done 3 times, one for each implementation (BTC-*, XRP, ETH-*),  we must test it on at least these 3 family of coins
- we need to test things properly by potentially mocking in the code the part that load the fees. (i've developed the feature mainly by modifying api/Fees.js. `promise = Promise.reject(new Error(..))` and `promise = new Promise(()=>{})` for respectively the error and the loading cases.
- transaction obviously should still work on these 3 families with the correct fees i've inputed.
- test that Custom fees inputing still work (in all scenarios)
- test that one of the preference (High/Medium/Low) still properly work

### CODE REVIEW

File: src/components/FeesField/BitcoinKind.js [hunk 1]
```diff
@@ -137,6 +145,8 @@ class FeesField extends Component<OwnProps, State> {
           value={feePerByte}
           onChange={onChange}
           onChangeFocus={this.onChangeFocus}
+          loading={!feePerByte && !error}
+          error={!feePerByte && error ? new FeeNotLoaded() : null}
```
[the remark this answers is not in the document]
Comment (gre) [reply src/components/FeesField/BitcoinKind.js]: yes.

typically we want to remap all errors (from the withFees btw) to this generic error message. if it's a Network is down case, we don't really care. the FeeNotLoaded says there is a problem.. maybe we can improve in future.

**Comment by gre** [comment]
just waiting the wording feedback now. for the QA test i suggest we do all the tests in batch once everything is on develop.
